roles/openshift_health_checker/test/package_version_test.py

```diff
@@ -4,7 +4,7 @@
 
 
 @pytest.mark.parametrize('openshift_release, extra_words', [
-    ('111.7.0', ["no recommended version of Open vSwitch"]),
+    # ('111.7.0', ["no recommended version of Open vSwitch"]),
```

Comment:
We can simply delete this line since the intended behavior now is that newer versions will automatically default to the last known version.